src/ui/Home.fs

```diff
@@ -6,4 +6,7 @@ open type Feliz.prop
 
 [<ReactComponent>]
 let Home () =
-    h1 [ text "Home"; className [ "text-3xl"; "font-bold"; "underline" ] ]
+    Html.div [
+        prop.style [ style.backgroundColor "#000000"; length.vh 100 |> style.height ]
```

Comment:
This should probably be a web-compliant colour name. Consider `black`?

Reply:
I made the background black just for testing convenience. I can change it to "black" though.